docs/index.md

```diff
@@ -1,639 +1,128 @@
-    wgo   Live reload for Go apps
-    qgo   Go client based on Qt5
-    gox   Dead simple, no frills Go cross compile tool
-    got   Version control system which prioritizes ease of use and simplicity over flexibility
-    gom   GObject to SQLite object mapper
-    gol   Command-line utility for creating and managing Geographic Object Libraries
-    goa   Design-based APIs and microservices in Go
-    ego   Run Linux desktop applications under a different local user
-    wego  Weather app for the terminal
-
-    Showing 10 of 4772 results. Use `flox search go --all` to see the full list.
-
-    Use 'flox show <package>' to see available versions
-    ```
-
-=== "Rust"
-
-    Let's do a search for `rust` and see what we get:
-    ```
-    my_project $ flox search rust
-    rustc    Safe, concurrent, practical language (wrapper script)
-    irust    Cross Platform Rust Repl
-    thrust   Chromium-based cross-platform / cross-language application framework
-    rustus   TUS protocol implementation in Rust
-    rustup   Rust toolchain installer
-    rustic   fast, encrypted, deduplicated backups powered by pure Rust
-    mrustc   Mutabah's Rust Compiler
-    rustfmt  Tool for formatting Rust code according to style guidelines
-    rustcat  Port listener and reverse shell
-    ht-rust  Friendly and fast tool for sending HTTP requests
-
-    Showing 10 of 810 results. Use `flox search rust --all` to see the full list.
-
-    Use 'flox show <package>' to see available versions
-
-    Related search results for 'cargo':
-    cargo     Downloads your Rust project's dependencies and builds your project
-    cargo-c   Cargo subcommand to build and install C-ABI compatible dynamic and static libraries
-    cargo-ui  GUI for Cargo
-    ```
-
-    You'll notice that there's not a `rust` package!
-    The Rust toolchain is unbundled in the Flox Catalog,
-    so you'll instead want `rustc`, `cargo`, etc.
-    For more details about setting up a Rust environment,
-    see the [Rust cookbook page][rust-cookbook].
+### __Cross-platform package management for your whole system__
 
----
+Set up your [default environment](https://flox.dev/docs/tutorials/default-environment/) with a set of 
+packages that you always want available, whether you're on macOS or Linux—x86 or ARM.
+
+Then, when you need to [set up a new laptop](https://flox.dev/blog/setting-up-a-new-laptop-made-easy-with-flox/
+), or [keep multiple machines in sync](https://flox.dev/docs/tutorials/sharing-environments/#always-using-the-same-environment-across-multiple-devices
+), you can be sure you're using the exact same versions, no matter when or where you need them. 
+
+If you're already using Homebrew, you can easily [migrate or use Homebrew and Flox together](https://flox.dev/docs/tutorials/migrations/homebrew/).
+
+
+
+### __Consistent builds from local to CI to production__
+
+Flox lets you define what an environment _is_ in a way that can be reused across local dev, CI, and production. 
```

Comment:
I think you could drop this sentence.

Reply:
Let's leave it in. Unless @barstoolbluz would you cut it or rewrite?